packages/expo-sharing/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ### 🐛 Bug fixes
 
+- On iOS, dismiss share sheet after sharing to an app is cancelled, so the file doesn't fail to attach when trying to share again ([#19656](https://github.com/expo/expo/pull/19656) by [@keith-kurak](https://github.com/keith-kurak))
```

Comment:
See https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#spelling-and-word-choices 😄 
```suggestion
- On iOS, dismiss share sheet after sharing to an app is canceled, so the file doesn't fail to attach when trying to share again ([#19656](https://github.com/expo/expo/pull/19656) by [@keith-kurak](https://github.com/keith-kurak))
```